usaspending_api/etl/subaward_etl.py

```diff
@@ -68,6 +65,7 @@ def load_subawards(submission_attributes, db_cursor):
         # a matching parent award id, piid, and submission attributes
         award = Award.objects.filter(transaction__submission=submission_attributes,
                                      transaction__contract_data__piid=row['piid'],
+                                     transaction__contract_data__isnull=False,
```

Comment:
Nope! OneToOne relationships do this weird thing where even though contract_data is null, it still lets you check contract_data__piid, and returns null. Just making sure to be explicit here.